### DISCUSSION TITLE
Engineering Lead, Games - PBS KIDS

### DISCUSSION THREAD

**Discussion by bigtimebuddy** [pixijs/pixijs#7147] [Jobs]
There's an exciting opportunity to work with the PBS KIDS [Games App](https://apps.apple.com/us/app/pbs-kids-games/id1050773989) as an [Engineering Lead](https://www.pbs.org/about/careers/current-openings/?gnk=job&gni=8a7883a8766d9a53017690658f4f3c48&lang=en). While this isn't a PixiJS gig _specifically_, this organization uses PixiJS heavily in their games and has been a major contributor to this open source project.

I have worked with PBS KIDS in the past and they are a great organization filled with many excellent people. This role also would work on developing [SpringRoll](https://github.com/springroll/springroll) another project I helped start in collaboration with PBS KIDS.

**Comment by GoodBoyDigital** [comment]
Yeah, these guys are great! Highly recommend going for this one! (I work with them on games app too at the mo!)
